src/gossip/emitter.go

```diff
@@ -202,11 +202,11 @@ func (em *Emitter) addTxs(e *inter.Event, poolTxs map[common.Address]types.Trans
 	maxGasUsed := em.maxGasPowerToUse(e)
 
 	now := time.Now()
-	members := em.world.Engine.GetMembers()
-	membersArr := members.SortedAddresses() // members must be sorted deterministically
-	membersArrStakes := make([]pos.Stake, len(membersArr))
-	for i, addr := range membersArr {
-		membersArrStakes[i] = members[addr]
+	validators := em.world.Engine.GetValidators()
+	validatorsArr := validators.SortedAddresses() // validators must be sorted deterministically
+	validatorsArrStakes := make([]pos.Stake, len(validatorsArr))
+	for i, addr := range validatorsArr {
+		validatorsArrStakes[i] = validators[addr]
```

Comment:
looks like the logic here should be moved to some where earlier. If addTxs is called twice on the same event, the code becomes repetitive.
Presumably, validator set and their stakes are fixed for the whole epoch. simpler.